index.js

```diff
@@ -1 +1,67 @@
 'use strict';
+var path            = require('path');
+
+var gulp            = require('gulp');
+var pluginRegistry  = require('plugin-registry');
+
+var taskYargsRun    = require('./lib/util/task-yargs-run');
+
+var defaultTaskPlugins = [
+  'html',
+  'css',
+  'javascript',
+  'test',
+  'build',
+  'release',
+  'server',
+  'watch',
+  'init',
+  'webstorm'
+].map(function parseDefaultPluginDefinition(taskName) {
+  return {
+    name: taskName,
+    requirePath: path.resolve(__dirname, 'tasks', taskName),
+    category: 'task',
+    isDefault: true
+  };
+});
+
+var angularityJson;
+try {
+  angularityJson = require(path.resolve('angularity.json'));
+}
+catch (ex) {
+  // Do nothing
+}
+var configTaskPlugins = (angularityJson && angularityJson.plugins) || [];
+if (configTaskPlugins.constructor !== Array) {
```

Comment:
In this case, not really, because it is guaranteed to always be an array primitive. I'd use `_.merge` if I were unsure whether the var extends the native array, or when a polyfill is necessary